### DISCUSSION TITLE
the icon in safari

### DISCUSSION THREAD

**Discussion by Kanbebetter** [quoid/userscripts#542] [General]
could anyone please add a feature that can change the defalult icon in safari app, or just change the icon in safari, I want the icon to be more rounded or more exquisite.

**Comment by ACTCD** [comment]
The [icons](https://github.com/quoid/userscripts/tree/main/xcode/Safari-Extension/Resources/images) are packaged with the assets and there doesn't seem to be a way to replace them after the build.

But if you plan to build the extension yourself, you can always replace them before building.

We plan to use new SVG icons in the next major release, but have no estimated timeframe.
